### ISSUE TITLE
fails on the new Flutter 3.10

### ISSUE THREAD

**Issue by ahndwon** [polarby/render#14]
**Describe the bug**
windows is missing on Flutter 3.10
```
:276:9: Error: No named parameter with the name 'window'.
        window: ui.window,
        ^^^^^^
/opt/homebrew/Caskroom/flutter/2.10.0/flutter/packages/flutter/lib/src/rendering/view.dart:68:3: Context: Found this candidate, but the arguments don't match.
  RenderView({
  ^^^^^^^^^^
```
**To Reproduce**
1. Upgrade to Flutter 3.10
2. Build

**Expected behavior**
Build

**Comment by polarby** [comment]
Thanks for opening this issue. I am on it!

**Comment by polarby** [comment]
Fixed with release 1.3 (476e5fb480c9481620d4ccdd6c75c230aae2d184)
